drivers/I2CSlave.h

```diff
@@ -87,7 +92,10 @@ class I2CSlave {
      *
      *  @param hz The bus frequency in Hertz.
      */
-    void frequency(int hz);
+    void frequency(int hz)
```

Comment:
Same comment: many function implementations have been moved to .h. It should not have been changed.

Reply:
See my overall comment here: https://github.com/ARMmbed/mbed-os/pull/10955#issuecomment-509937556